### ISSUE TITLE
[Feature]: NuGet.org should show framework references

### ISSUE THREAD

**Issue by cremor** [NuGet/NuGetGallery#9615]
### Related Problem

I am trying to figure out if a specific package has a `<frameworkReference>`. I'm unable to see this on NuGet.org, instead I have to open the NuGet Package Explorer and go to the "Dependencies" tab there.

### The Elevator Pitch

The "Frameworks" tab on NuGet.org should also show framework references.

**Comment by bkoelman** [comment]
Yes, very confusing. It's pretty critical to know about framework references (related to https://github.com/json-api-dotnet/JsonApiDotNetCore/pull/2017).
